### DISCUSSION TITLE
Pencere vergisi sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#90] [Zeka Sorusu] [answered]
Hükümetimiz ekonomik krizi atlatmak için pencere vergisi çıkarmayı planlıyormuş.
Buna göre 0,35 m² pencereler için 830 lira, 0,45 m² pencereler için 930 lira, 0,55 m² pencereler için 1030 lira, 0,65 m² pencereler için de 1130 lira vergi ödenecekmiş.
1 m² pencereler için ne kadar vergi ödenir?

**Comment by FatihOnder81** [comment, marked as answer]
0.35 den itibaren her 0.10 luk artısta 100tlik fiyat artısı oluyor 0.65 1130 tl oluyorsa 1 olması için 0.35 fark var oda 350 tl yapar ve 1480 tl olur
